shared/src/main/kotlin/com/egm/stellio/shared/model/parameter/GeoQuery.kt

```diff
@@ -0,0 +1,119 @@
+package com.egm.stellio.shared.model.parameter
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.raise.either
+import arrow.core.right
+import com.egm.stellio.shared.model.APIException
+import com.egm.stellio.shared.model.BadRequestDataException
+import com.egm.stellio.shared.model.ExpandedEntity
+import com.egm.stellio.shared.model.ExpandedTerm
+import com.egm.stellio.shared.model.WKTCoordinates
+import com.egm.stellio.shared.model.parseGeometryToWKT
+import com.egm.stellio.shared.model.stringifyCoordinates
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.JsonLdUtils.JSONLD_VALUE
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_GEOPROPERTY_VALUE
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_LOCATION_PROPERTY
+import com.egm.stellio.shared.util.JsonLdUtils.expandJsonLdTerm
+import com.egm.stellio.shared.util.JsonUtils
+import com.egm.stellio.shared.util.decode
+
+data class GeoQuery(
```

Comment:
not clear to me why `GeoQuery` has been moved in this package but not `LinkedEntityQuery`, `PaginationQuery`,...

Reply:
more globally, this new `parameter` sub-package is a bit fuzzy.